projects/2015/Compare.vcxproj

```diff
@@ -201,7 +217,7 @@
       <SubSystem>Windows</SubSystem>
       <OptimizeReferences>true</OptimizeReferences>
       <EnableCOMDATFolding>true</EnableCOMDATFolding>
-      <ImportLibrary>$(OutDir)Compare.lib</ImportLibrary>
+      <ImportLibrary>ComparePlugin.lib</ImportLibrary>
```

Comment:
Could you please add `<PostBuildEvent></PostBuildEvent>` section for this build variant?
Thanks.